### ISSUE TITLE
Documentation: examples for regression

### ISSUE THREAD

**Issue by mapmeld** [keras-team/autokeras#1061]
I recently finished using AutoKeras on a regression task with tabular data. It worked great (thanks!!) but while working on it, I noticed the tutorial is empty ( https://autokeras.com/tutorial/structured_data_regression/ ) yet there is now an example in the documentation in this repo.  Is a new release planned?

In the meantime - there's still no example for text regression or image regression. I will try to add examples for one or both to fully close this issue, if no one else is working on it.

**Comment by haifeng-jin** [comment]
We currently are not working on these tutorials. It would be great if you can contribute any of them. Please follow the image classification tutorial style to write it. Thank you!

**Comment by anavanab99** [comment]
> I recently finished using AutoKeras on a regression task with tabular data. It worked great (thanks!!) but while working on it, I noticed the tutorial is empty ( https://autokeras.com/tutorial/structured_data_regression/ ) yet there is now an example in the documentation in this repo. Is a new release planned?
> 
> In the meantime - there's still no example for text regression or image regression. I will try to add examples for one or both to fully close this issue, if no one else is working on it.

My Structured Data Regression doesn't seem to work, how did you do it

**Comment by mapmeld** [comment]
What was your issue? I didn't check for accuracy on mine, but I can give you a code sample

```python
import autokeras as ak
from sklearn.model_selection import train_test_split

# X and y from Pandas dataframes
X = df.drop(columns=['host_id', 'price'])
y = df['price']

# split up if you're holding onto validation data
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.33)
y_arr = y_train.to_numpy().astype('int64') # this is now a 1D numpy array

clf = ak.StructuredDataRegressor(max_trials=10)
clf.fit(X_train, y_arr, validation_data=(X_test, y_test))
```
